modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -0,0 +1,72 @@
+= What's New in DataWeave
+
+The 2.4 version of DataWeave introduces the following features:
+
+* Larger than memory strings.
```

Comment:
Can we point to some of the details here?

Reply:
TODO: yes, that doc is in progress. I'll link to it once I know the file name. https://www.mulesoft.org/jira/browse/DOCS-12018

Reply:
```suggestion
* Ability to read larger-than-memory strings by setting the indexed reader property `com.mulesoft.dw.max_memory_allocation` and optionally, the performance-related property `com.mulesoft.dw.buffered_char_sequence.enabled`. See xref:dataweave-formats.adoc[Supported Data Formats].
```

Reply:
This could be more accurate: 
Ability to read larger-than-memory strings. 
When you are using the indexed reader strategy and processing a `String` with a size larger than 1.5 MB, DataWeave automatically splits the value in chunks to avoid out-of-memory issues. See xref:dataweave-formats.adoc[Supported Data Formats].

Reply:
@mlischetti Updated the suggestion with your description while keeping the new configuration properties in it. Does this work?

Reply:
Yes, it works. 
One additional note:  com.mulesoft.dw.max_memory_allocation is not a new reader properties. 
So we can: 
1- Remove the whole "Reader reader properties..." section 
2- Just say, "Related reader properties are `com.mulesoft.dw.max_memory_allocation` and `com.mulesoft.dw.buffered_char_sequence.enabled`"

Reply:
ok, opted for #2. removed "are new"